ecms_base/features/custom/ecms_hotel/config/install/field.storage.media.field_media_image_2.yml

```diff
@@ -0,0 +1,29 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - file
+    - image
+    - media
+id: media.field_media_image_2
+field_name: field_media_image_2
```

Comment:
Same media field name as in PR #28. I'm not sure if there could potentially be a conflict?